receiver/redisreceiver/metric_functions.go

```diff
@@ -117,7 +117,7 @@ func connectedClients() *redisMetric {
 	return &redisMetric{
 		key:    "connected_clients",
 		name:   "redis/clients/connected",
-		mdType: metricspb.MetricDescriptor_GAUGE_INT64,
+		pdType: pdata.MetricDataTypeIntGauge,
```

Comment:
This should be a non-monotonic IntSum correct? It is still a sum even though is not always going up. Our data model is a bit different than others because we have monotonic Sum (what others call counters, always growing) and non-monotonic Sum (can go up/down).

Same applies to all the metrics that logically are sums of some measurements.

Reply:
Ah ok, thanks for the explanation. I have changed from gauge to non-monotonic sum: `connected_clients`, `blocked_clients`, `rdb_changes_since_last_save`, and `connected_slaves`.